testdata/xml-data/circular-references/analyse_circles_in_xml.py

```diff
@@ -37,8 +37,8 @@ def analyse_circles_in_data(xml_filepath: Path, tracer_output_file: str, save_tr
     print(f"Total Number of resptr Links: {len(resptr_links)}")
     print(f"Total Number of XML Texts with Links: {len(xml_links)}")
     print("=" * 80)
-    g, node_index_lookup, edges, node_indices = make_graph(resptr_links, xml_links, all_resource_ids)
-    _, _, stash_counter = generate_upload_order(g, node_index_lookup, edges, node_indices)
+    graph, rustworkx_index_to_id, edges = make_graph(resptr_links, xml_links, all_resource_ids)
+    _, _, stash_counter = generate_upload_order(graph, rustworkx_index_to_id, edges)
```

Comment:
it is convention in many graph processing libraries to call the graph variable `g`

Reply:
ah okay, I didn't know that. For me, `g` was difficult to understand.